client/state/action-types.ts

```diff
@@ -33,6 +33,12 @@ export const ACTIVE_PROMOTIONS_RECEIVE = 'ACTIVE_PROMOTIONS_RECEIVE';
 export const ACTIVE_PROMOTIONS_REQUEST = 'ACTIVE_PROMOTIONS_REQUEST';
 export const ACTIVE_PROMOTIONS_REQUEST_FAILURE = 'ACTIVE_PROMOTIONS_REQUEST_FAILURE';
 export const ACTIVE_PROMOTIONS_REQUEST_SUCCESS = 'ACTIVE_PROMOTIONS_REQUEST_SUCCESS';
+export const ACTIVITY_LOG_RETENTION_POLICY_REQUEST = 'ACTIVITY_LOG_RETENTION_POLICY_REQUEST';
```

Comment:
Did you consider using a newer pattern like React Query ( p4TIVU-9AF-p2 )? The redux state pattern is perfectly fine, imo, but I think the newer patterns involve much less boilerplate.

Reply:
To be completely honest, I wasn't aware React Query had been blessed for use in Calypso yet. 😅 I'll keep that in mind for future PRs, though!